reporting/miro_transformer/src/test_miro_pipeline.py

```diff
@@ -0,0 +1,69 @@
+import json
+import miro_transformer
+from reporting_pipeline.test_pipeline import given_s3_has, create_sns_message
+
+
+def create_miro_hybrid_data(raw_data):
+    return {
+        "MiroCollection": "images-2",
+        "data": raw_data,
+        "id": "miro/A0000001",
+        "sourceId": "A0000001",
+        "sourceName": "miro",
+        "version": 1,
+    }
+
+
+def raw_data():
+    return """{
+                \"an_array\": [
+                    \"a value\",
+                    \"another value\"
+                ],
+                \"an_array_of_date\": [
+                            \"22/04/2017\",
+                            \"19/12/2011\"],
+                \"string_field\": \"a string\",
+                \"null_field\": null,
+                \"single_date\": \"24/04/2004\",
+                \"dict_field\": {
+                      \"a\": \"a_value\"
+                  },
+                \"image_source\": \"ignored field\"
+            }"""
```

Comment:
Note, for future reference: an alternative would be `json.dumps(json.dumps(my_dict))`, which might be a little easier to read.